packages/helpers/src/index.ts

```diff
@@ -375,6 +376,59 @@ export function createTransactionFromSkeleton(
   return tx;
 }
 
+type Promisable<T> = Promise<T> | T;
+export type LiveCellFetcher = (outPoint: OutPoint) => Promisable<Cell>;
+
+/**
+ * create a {@link TransactionSkeleton} from a {@link Transaction}
+ * @example
+ * ```js
+ * const fetcher = new RPC('localhost:8114').getLiveCell
```

Comment:
```suggestion
 * const rpc = new RPC('localhost:8114')
 * const fetcher = (outPoint: OutPoint) => rpc.getLiveCell(outPoint)
```

Direct use of the RPC method may result in loss of `this` context